front/src/reducers/osrdconf/types.ts

```diff
@@ -53,6 +54,10 @@ export type OperationalStudiesConfState = OsrdConfState & {
   powerRestriction: PowerRestriction[];
   timeWindow: Duration;
   interval: Duration;
+  addedExceptions: {
+    key: string;
+    startTime: Date;
```

Comment:
As we don't perform any conversion from `startTime` (front format) to `start_time` (editoast format), we should use the editoast syntax directly otherwise we get a white screen when adding the paced train.
```suggestion
    start_time: Date;
```

Reply:
I think we need to transform the data anyways: https://github.com/OpenRailAssociation/osrd/pull/11848#discussion_r2137406122

In which case, we can keep `startTime` as-is.

Reply:
fixed. as discussed, now doing the transformation in `formatPacedTrainPayload`